packages/tables/src/Columns/Text.php

```diff
@@ -111,4 +113,11 @@ public function url($url)
 
         return $this;
     }
+    
+    public function shouldOpenInNewTab()
```

Comment:
I think the method name should be `openInNewTab` - `shouldOpenInNewTab` sounds like it returns a boolean, instead of instructing the field to open in a new tab.

Reply:
I agree. And second param in URL is nice too - making the changes now.